src/Fhir.Anonymizer.Core.UnitTests/Processors/RedactProcessorTests.cs

```diff
@@ -94,7 +94,7 @@ public void GivenAPostalCodeNode_WhenRedact_RedactedNodeShouldBeReturn()
             node = ElementNode.FromElement(new FhirString("54321").ToTypedElement());
             node.Name = "postalCode";
             processor.Process(node);
-            Assert.Equal("00000", node.Value.ToString());
+            Assert.Equal(string.Empty, node.Value.ToString());
```

Comment:
Can we make it consistant to represent empty string as string.Empty? Like in Line 25, 40 etc.

Reply:
Thanks for pointing this out. The value is updated to null due to another change in PR #16.